pkg/storage/leveldb/leveldb_store.go

```diff
@@ -112,7 +112,7 @@ type leveldbStore struct {
 	db *leveldb.DB
 }
 
-// Put stores the key and the record
+// Store stores the key and the record
```

Comment:
Again - comment updated but not method? In a few places actually.

Reply:
done